### DISCUSSION TITLE
Enhancement: include sanic in benchmark

### DISCUSSION THREAD

**Discussion by devkral** [litestar-org/litestar#2169] [Q&A] [answered]
### Summary

It would be interesting to see how fast this project is compared to sanic. Can you include in the benchmarks sanic?

---

## Funding
* If you would like to see an issue prioritized, make a pledge towards it!
* We receive the pledge once the issue is completed & verified

<a href="https://polar.sh/litestar-org/litestar/issues/2168">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/litestar-org/litestar/issues/2168/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/litestar-org/litestar/issues/2168/pledge.svg">
</picture>
</a>

**Comment by Alc-Alc** [comment, marked as answer]
Hey, just as a side note, in case you missed it, it seems that Sanic is incorporated here: https://github.com/litestar-org/api-performance-tests.

I'm not opposed to including this in the main documentation, I will let the maintainers weigh in on that. I just wanted to let you know.

**Reply by JacobCoffee** [reply]
We will be updating benchmarks when 2.0 reaches stable soon, so it should be included in the run.

**Comment by devkral** [comment]
thanks, missed that. Are there results?

**Comment by Alc-Alc** [comment]
> thanks, missed that. Are there results?

I don't think so. In the mean time, it should be fairly trivial to run it though 🙂

**Comment by Goldziher** [comment]
> thanks, missed that. Are there results?

there are in the api-benchmarks repository. You can see there.
